docs/release_notes.adoc

```diff
@@ -8,12 +8,12 @@ include::include.adoc[]
 === Groovy-3.0 Support
 
 The main feature of this milestone is support for Groovy 3.
-To use Spock in your Groovy 3 project just select the `spock-\*-2.0-M2` artifact(s) ending with `-groovy-3.0`.
+To use Spock in your Groovy 3 project just select the `spock-*-2.0-M2` artifact(s) ending with `-groovy-3.0`.
```

Comment:
the escaping of  `*` was necessary, otherwise it rendered as bold, at least in the intellj preview, and the output looks ok

Reply:
urgh, it seems asciidoc is rather ugly here, it is necessary inside the note as there are other `*` around that one, but standalone it doesn't escape the `*` but renders as `\*`

Reply:
Yes, in one place it was needed, in the another not...